config/locales/en.yml

```diff
@@ -11,6 +11,7 @@ en:
             redirect_uri:
               fragment_present: 'cannot contain a fragment.'
               invalid_uri: 'must be a valid URI.'
+              unspecified_scheme: 'a scheme must be specified.'
```

Comment:
Humanized errors will look strange, like `Redirect URI a scheme must be specified.`

Let's make it `must specify a scheme` so that result error message will be `Redirect URI must specify a scheme`